src/app/shared/entry-actions/entry-actions.service.ts

```diff
@@ -111,12 +111,14 @@ export class EntryActionsService {
   openNoDefaultDialog(entry: Entry, entryType: string, showVersions: EventEmitter<void> | null): void {
     const informationDialogData: InformationDialogData = {
       title: 'Default Version Required',
-      message: `Your ${entryType} must have a default version to be published.  Please use the the Actions menu in the Versions tab to select a default version.`,
+      message: `Your ${entryType} must have a default version to be published.  Please use the Actions menu in the Versions tab to select a default version.`,
       closeButtonText: 'OK',
     };
     const observable = this.informationDialogService.openDialog(informationDialogData, bootstrap4mediumModalSize);
     if (showVersions != null) {
-      observable.subscribe(() => { showVersions.emit(); });
+      observable.subscribe(() => {
+        showVersions.emit();
+      });
```

Comment:
these lines were changed likely by the auto commit formatting hooks, no functional changes related to this PR.